examples/affinity/README.md

```diff
@@ -0,0 +1,102 @@
+
+```console
+oc get nodes --show-labels
+```
+
+In this cluster, we want the `coffeeshop-frontend` and `coffeeshop-backend` applications to be co-located on the same nodes with SSD storage types.
+
+Here is a YAML snippet of the `RuntimeComponent` custom resource (CR) for the `coffeeshop-backend` application with two replicas. The custom resource has `.spec.affinity.podAntiAffinity` configured to ensure the scheduler does not co-locate pods for the application on a single node. As described in the [user guide](https://github.com/application-stacks/runtime-component-operator/blob/master/doc/user-guide.adoc#labels), the application pods are labelled with `app.kubernetes.io/instance: metadata.name`.
+
+The custom resource also has `spec.affinity.nodeAffinityLabels` to ensure the application pods will run on nodes with the `disktype: ssd` label.
+
+```yaml
+apiVersion: app.stacks/v1beta1
+kind: RuntimeComponent
+metadata:
+  name: coffeeshop-backend
+spec:
+  applicationImage: 'k8s.gcr.io/pause:2.0'
+  replicas: 2
+  affinity:
+    nodeAffinityLabels:
+      disktype: ssd
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchExpressions:
+              - key: app.kubernetes.io/instance
+                operator: In
+                values:
+                  - coffeeshop-backend
+          topologyKey: kubernetes.io/hostname
+```
+
+The YAML snippet below shows the `RuntimeComponent` CR for the `coffeeshop-frontend` application. The snippet has the `.spec.affinity.podAffinity` configured which informs the scheduler that all its replicas are to be co-located with pods that have selector label `app.kubernetes.io/instance: coffeeshop-backend`. The CR also includes `.spec.affinity.podAntiAffinity` to ensure that each `coffeeshop-frontend` replica does not co-locate on a single node. In addition, the `spec.affinity.nodeAffinityLabels` parameter is specified to ensure the application pods will run on nodes with the `disktype: ssd` label.
+
+```yaml
+apiVersion: app.stacks/v1beta1
+kind: RuntimeComponent
+metadata:
+  name: coffeeshop-frontend
+spec:
+  applicationImage: 'k8s.gcr.io/pause:2.0'
+  replicas: 2
+  affinity:
+    nodeAffinityLabels:
+      disktype: ssd
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchExpressions:
+              - key: app.kubernetes.io/instance
+                operator: In
+                values:
+                  - coffeeshop-frontend
+          topologyKey: kubernetes.io/hostname
+    podAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchExpressions:
+              - key: app.kubernetes.io/instance
+                operator: In
+                values:
+                  - coffeeshop-backend
+          topologyKey: kubernetes.io/hostname
+```
+
+To see where the which nodes the pods are running, run the following command:
```

Comment:
This sentence needs an update.

Reply:
Addressed in https://github.com/application-stacks/runtime-component-operator/pull/133/commits/06f4c1014e6f595740b3a94c26770547e0fd7cf7